examples/advanced/lr-newton-raphson/README.md

```diff
@@ -0,0 +1,253 @@
+  which weighs the contribution from each client based on the number
+  of local training samples. The aggregated Newton-Raphson update is
+  returned as an `FLModel`.
+- After getting the aggregated Newton-Raphson update, an
+  [`update_model()`](job/newton_raphson/app/custom/newton_raphson_workflow.py#L172)
+  method is implemented to actually apply the Newton-Raphson update to
+  the global model.
+- The last step is to save the updated global model, again through
+  the `NewtonRaphsonModelPersistor` using `save_model()`.
+
+
+On the client side, the local training logic is implemented
+[here](job/newton_raphson/app/custom/newton_raphson_train.py). The
+implementation is based on the [`Client
+API`](https://nvflare.readthedocs.io/en/main/programming_guide/execution_api_type.html#client-api). This
+allows user to add minimum `nvflare`-specific codes to turn a typical
+centralized training script to a federated client side local training
+script.
+- During local training, each client receives a copy of the global
+  model, sent by the server, using `flare.receive()` API. The received
+  global model is an instance of `FLModel`.
+- A local validation is first performed, where validation metrics
+  (accuracy and precision) are streamed to server using the
+  [`SummaryWriter`](https://nvflare.readthedocs.io/en/main/apidocs/nvflare.client.tracking.html#nvflare.client.tracking.SummaryWriter). The
+  streamed metrics can be loaded and visualized using tensorboard.
+- Then each client computes it's gradient and Hessian based on local
+  training data, using their respective theoretical formula described
+  above. This is implemented in the
+  [`train_newton_raphson()`](job/newton_raphson/app/custom/newton_raphson_train.py#L82)
+  method. Each client then sends the computed results (always in
+  `FLModel` format) to server for aggregation, using `flare.send()`
+  API.
+
+Each client site corresponds to a site listed in the data table above.
+
+A [centralized training script](./train_centralized.py) is also
+provided, which allows for comparing the federated Newton-Raphson
+optimization versus the centralized version. In the centralized
+version, training data samples from all 4 sites were concatenated into
+a single matrix, used to optimize the model parameters. The
+optimized model was then tested separately on testing data samples of
+the 4 sites, using accuracy and precision as metrics.
+
+Comparing the federated [client-side training
+code](job/newton_raphson/app/custom/newton_raphson_train.py) with the
+centralized [training code](./train_centralized.py), we can see that
+the training logic remains similar: load data, perform training
+(Newton-Raphson updates), and valid trained model. The only added
+differences in the federated code are related to interaction with the
+FL system, such as receiving and send `FLModel`.
+
+## Set Up Environment & Install Dependencies
+
+Create & activate a virtual environment
```

Comment:
Should we be following or linking to the instructions [here](https://github.com/NVIDIA/NVFlare/tree/main/examples#set-up-a-virtual-environment)?

Reply:
I actually forgot that we have already instructions for setting up virtual env for flare examples.
Just changed the README to link to the instructions. 
Also removed the line for installing falre in requirements.txt.